testing/templates/for-destructoring-tuple.html

```diff
@@ -0,0 +1 @@
+{% for (((a,), b), c) in v %}{{a}}{{b}}{{c}}-{% endfor %}
```

Comment:
For short templates like this, maybe put them inline with `#[template(source)]`?